tf_explain/core/vanilla_gradients.py

```diff
@@ -1,13 +1,29 @@
 """
 Core Module for Vanilla Gradients
 """
+from warnings import warn
 import tensorflow as tf
 
 from tf_explain.utils.display import grid_display
 from tf_explain.utils.image import transform_to_normalized_grayscale
 from tf_explain.utils.saver import save_grayscale
 
 
+UNSUPPORTED_ARCHITECTURE_WARNING = "\
+Unsupported model architecture for VanillaGradients. The last two layers of \
+the model should be: a layer which computes class scores with no activation, \
+followed by an activation layer."
```

Comment:
It might be a personal opinion, but I'm not a fan of the line breakers. Could you format it this way instead?

```python
UNSUPPORTED_ARCHITECTURE_WARNING = (
    "Unsupported model architecture for VanillaGradients. The last two layers of "
    "the model should be: a layer which computes class scores with no activation, "
    "followed by an activation layer."
)
```

Reply:
Ah, I didn't know you could do it that way!